WordPress/Classes/Utility/BackgroundTasks/WeeklyRoundupBackgroundTask.swift

```diff
@@ -158,33 +224,40 @@ private class WeeklyRoundupDataProvider {
             return
```

Comment:
The `result` closure before the return statement should be called from the `context` queue too?

Reply:
See https://github.com/wordpress-mobile/WordPress-iOS/pull/21076#discussion_r1267095822

Reply:
Resolving this conversation as this change is no longer needed. See this [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/21076#discussion_r1267526635).